sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -50,7 +50,7 @@ Netty and include OkHTTP client in your pom.xml.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-beta.1</version>
```

Comment:
This, unfortunately, needs to be left at 1.0.0 which is the released version.

Reply:
fixed